snakemake-local-run/inputs.yaml

```diff
@@ -0,0 +1,5 @@
+events: 20000
```

Comment:
We don't need to push this file. Everything inside `snakemake-local-run` is temporary to run the workflow locally.